intro-HPC/ch_running_jobs_with_input_output_data.tex

```diff
@@ -364,6 +364,32 @@ \subsection{Your UGent home drive and shares \label{subsec:ugent-shares}}
 
 Each hour the process will check if your ticket should be renewed.
 
+\subsection{UGent shares with globus \label{subsec:globus-ugent-shares}}
+
+In order to access your UGent home and shares inside the globus access point, you'll first have to authenticate. To do this, you have to ssh to the globus endpoint from a loginnode. You will be prompted for your UGent username and password to authenticate:
```

Comment:
`access point` -> `endpoint`.
"you'll first have to authenticate" -> "you first have to generate authentication credentials on the endpoint"
line too long.